src/uid2Sdk.ts

```diff
@@ -148,16 +123,26 @@ export abstract class UID2SdkBase {
     return this._tokenPromiseHandler.createMaybeDeferredPromise(token ?? null);
   }
 
+  // Deprecated
```

Comment:
Is it worth using JSDoc to make this more clear?

```
/**
 * Deprecated
 */
```

![image](https://github.com/IABTechLab/uid2-web-integrations/assets/137864392/a26f4963-14d5-437e-91f5-fdff56ccc6ad)